openfe/setup/atom_mapping/lomap_scorers.py

```diff
@@ -198,15 +194,13 @@ def hybridization_score(mapping: LigandAtomMapping, beta=0.15):
         if mismatch:
             nmismatch += 1
 
-    hybridization_rule = math.exp(- beta * nmismatch)
-
-    return 1 - hybridization_rule
+    return math.exp(- beta * nmismatch)
 
 
-def sulfonamides_score(mapping: LigandAtomMapping, beta=0.4):
+def sulfonamides_score(mapping: LigandAtomMapping, beta=0.4) -> float:
     """Checks if a sulfonamide appears and disallow this.
 
-    Returns (1 - math.exp(- beta)) if this happens, else 0
+    Returns math.exp(- beta) if this happens, else 1.0
```

Comment:
```suggestion
    Returns ``math.exp(- beta)`` if a sulfonamide group is mutated in or out, 
    otherwise 1.0. Testing has shown that growing a sulfonamide from scratch 
    performs very badly.

    Parameters
    ==========
    beta
        A positive float describing how much to penalise a growing sulfonamide.
        Smaller values indicate exponentially larger penalties. 
```

I took the "testing has shown" line from https://github.com/OpenFreeEnergy/Lomap/blob/822bf739d70d46575be8e915809990f1d479029a/lomap/mcs.py#L990 as the purpose of this scorer was not at all clear to me.